source/crud/tests/unified/countDocuments-comment.yml

```diff
@@ -0,0 +1,70 @@
+description: "count"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false # Avoid setting fail points with multiple mongoses
+      uriOptions: { retryReads: false } # Avoid retrying fail points with closeConnection
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name count-test
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
```

Comment:
If we look at `insertMany-comment.yml`, I see we did have a test with `maxServerVersion: 4.2.99` entitled "insertMany with comment - pre 4.4" that expects a server-side error. Can you add a similar test here?

Reply:
Actually, `aggregate` supports string comments down to 3.6 - https://www.mongodb.com/docs/manual/reference/command/aggregate/ Tests to comments in `aggregate.yml` have same `runOnRequirements`.
I am not sure if we should add new tests for pre 3.6 servers.

Reply:
I agree with not testing a pre-3.6 server. Having a test for a pre-4.4 server with a non-string comment seems reasonable, though.

Reply:
Spec added.